Formula/android-ndk.rb

```diff
@@ -46,4 +45,10 @@ def caveats; <<-EOS.undent
       #{prefix}/docs
     EOS
   end
+
+  test do
+    (testpath/"test.c").write("int main() { return 0; }")
+    cc =  `/usr/local/opt/android-ndk/ndk-which gcc`.strip
```

Comment:
Can't assume `/usr/local`. Looks like `ndk-which` is in `bin`, in which case you can just do:
```ruby
cc = Utils.popen_read("#{bin}/ndk-which gcc")
```
Or something similar..